src/clj/ataru/files/file_store.clj

```diff
@@ -23,9 +23,19 @@
           (json/parse-string true)
           (dissoc :version :deleted)))))
 
-(defn delete-file [key]
-  (let [url  (str (get-in config [:liiteri :url]) "/api/files/" key)
+(defn delete-file [file-key]
+  (let [url  (str (get-in config [:liiteri :url]) "/api/files/" file-key)
         resp @(http/delete url)]
-    (clojure.pprint/pprint resp)
+    (when (= (:status resp) 200)
+      (json/parse-string (:body resp) true))))
+
+(defn get-metadata [application-keys]
+  (let [query-part (->> application-keys
+                        (map-indexed (fn [idx key]
```

Comment:
Same functionality can be found at ataru.virkailija.application.handlers.cljs:183